src/components/header_menu.jsx

```diff
@@ -1,11 +1,81 @@
-import React from 'react'
-
-const HeaderMenu = props => (
-  <header className={'page-header-menu ' + (props.className || '')}>
-    <div className="page-header-menu--left">{props.left}</div>
-    {props.children}
-    <div className="page-header-menu--right">{props.right}</div>
-  </header>
-)
+import React, { useMemo } from 'react'
+
+import AppLinker from 'cozy-ui/transpiled/react/AppLinker'
+import AppIcon from 'cozy-ui/transpiled/react/AppIcon'
+import Divider from 'cozy-ui/transpiled/react/MuiCozyTheme/Divider'
+import Link from 'cozy-ui/transpiled/react/Link'
+import { SharedRecipients } from 'cozy-sharing'
+import { Typography } from '@material-ui/core'
+import { Slugs } from 'constants/strings'
+import { NotePath } from './notes/List/NotePath'
+import { CozyFile } from 'cozy-doctypes'
+import { getDriveLink } from 'lib/utils'
+
+import styles from './header_menu.styl'
+import { WithBreakpoints } from './notes/List/WithBreakpoints'
+import { Breakpoints } from 'types/enums'
+
+const HeaderMenu = ({
+  homeHref,
+  backFromEditing,
+  editorCorner,
+  isPublic,
+  file,
+  client
+}) => {
+  const drivePath = useMemo(
+    () => getDriveLink(client, file.attributes.dir_id),
+    [client, file.attributes.dir_id]
+  )
+
+  const { filename } = CozyFile.splitFilename(file.attributes)
+
+  return (
+    <header className={styles['header-menu']}>
```

Comment:
We should avoid creating custom css. Can't we use our utilities? https://docs.cozy.io/cozy-ui/styleguide/section-utilities.html

Reply:
I am going to dig into that but I can't guarantee I will find everything I need to avoid CSS

Reply:
After looking into it there's too much CSS I don't think I can translate it in atomic classes. Probably would be best to make a cozy-ui component (this toolbar is used by 3 apps). Should I create an issue?